app/controllers/application_controller.rb

```diff
@@ -2,4 +2,57 @@ class ApplicationController < ActionController::Base
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
+
+  def build_cities
```

Comment:
all these methods look really great, but when thinking about MVC, where's the most appropriate place for them?